ocp_resources/virtual_machine_instance.py

```diff
@@ -204,11 +205,39 @@ def get_xml(self):
         Returns:
             xml_output(string): VMI XML in the multi-line string
         """
+        cmd = f"virsh {self.virt_launcher_pod_hypervisor_connection_uri} dumpxml {self.namespace}_{self.name}"
         return self.virt_launcher_pod.execute(
-            command=["virsh", "dumpxml", f"{self.namespace}_{self.name}"],
+            command=shlex.split(cmd),
             container="compute",
         )
 
+    @property
+    def virt_launcher_pod_user(self):
```

Comment:
Do we require the UID, or can we make this simply `is_virt_launcher_pod_[root|user]`?

Reply:
I am checking for UID 107 in tests , which is QEMU UID.
For NonRoot VirtLauncher Pod it's QEMU UID

Reply:
Please rename 
```suggestion
    def virt_launcher_pod_user_uid(self):
```